single_tech_samples/datafactory/sample2_cicd_auto_publish/README.md

```diff
@@ -0,0 +1,322 @@
+# Automated Publish and CI/CD for Azure Data Factory
+
+This sample demonstrates the deployment of Azure Data Factory using Automated Publish Method. Usually, the Azure Data Factory deployment required Manual Publish setup where the developer had to manually Publish the ADF changes from the portal. This step generated the ARM Templates which were used in deployment step.
+
+In this sample, we have eliminated the Manual Publish step by using ADF Packages for Automated Publish. Now, the ARM templates will be generated automatically without any manual intervention.
+
+## ADF Introduction
+
+Azure Data Factory(ADF) is a service in Microsoft Azure that allows the developers to integrate and transform their data using various processes. ADF has good integrations with multiple internal and external services.
+
+In ADF, CI/CD essentially means deploying the Data Factory components(Linked Service, Triggers, Pipelines, Datasets) to a new environment(ADF) automatically using Pipelines.
```

Comment:
the Data Factory components(Linked Service, Triggers, Pipelines, Datasets) => various data factory entities such as linked service, triggers, pipelines, datasets etc.

Reply:
new environment(ADF) => new environment

Reply:
Pipelines => Azure Pipelines (you can also add link to it https://learn.microsoft.com/en-us/azure/devops/pipelines/get-started/what-is-azure-pipelines?view=azure-devops)

Reply:
DOne